Blanball/Views/LabeledCheckBox/LabeledCheckBox.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  LabeledCheckBox.swift
+//  Blanball
+//
+//  Created by Nik Dub on 10.03.2023.
+//
+
+import UIKit
+
+class LabeledCheckBox: UIView {
+    
+    // MARK: - Internal properties -
+    
+    @Published private(set) var state: LabeledCheckBoxState
+    
+    // MARK: - Private properties -
```

Comment:
checkbox should be separate from label, just add ATapGesture to their container for single action

Reply:
Fixed in `Changes According PR#7`